Library/.rubocop.yml

```diff
@@ -125,5 +125,7 @@ Style/DotPosition:
   EnforcedStyle: trailing
 
 # makes diffs nicer
-Style/TrailingComma:
+Style/TrailingCommaInArguments:
+  EnforcedStyleForMultiline: comma
```

Comment:
We probably want `no_comma` for this. As trailing comma for arguments is syntax error for Ruby 1.8.